src/core/Group.cpp

```diff
@@ -190,15 +190,30 @@ QString Group::defaultAutoTypeSequence() const
 QString Group::effectiveAutoTypeSequence() const
 {
     QString sequence;
-    const Group* grp = this;
-    do {
-        if (grp->autoTypeEnabled() == Group::Disable) {
-            return QString();
-        }
-
-        sequence = grp->defaultAutoTypeSequence();
-        grp = grp->parentGroup();
-    } while (grp && sequence.isEmpty());
+    bool enableSet = false;
+
+    const Group* group = this;
+    // check if group exist (if the entry has a group)
+    if(group) {
```

Comment:
This check doesn't make sense, "this" cannot be null unless you are calling a null pointer to begin with. This check needs to go in the function that calls into the group class.